sixtyfps_runtime/corelib/graphics/fps_counter.rs

```diff
@@ -0,0 +1,70 @@
+/* LICENSE BEGIN
+    This file is part of the SixtyFPS Project -- https://sixtyfps.io
+    Copyright (c) 2021 Olivier Goffart <olivier.goffart@sixtyfps.io>
+    Copyright (c) 2021 Simon Hausmann <simon.hausmann@sixtyfps.io>
+
+    SPDX-License-Identifier: GPL-3.0-only
+    This file is also available under commercial licensing terms.
+    Please contact info@sixtyfps.io for more information.
+LICENSE END */
+/*!
+This module contains a simple helper type to measure the average number of frames rendered per second.
+*/
+
+use crate::timers::*;
+use std::cell::RefCell;
+use std::rc::Rc;
+
+/// Helper class that rendering backends can use to provide an FPS counter
+pub struct FPSCounter {
+    frame_times: RefCell<Vec<instant::Instant>>,
+    update_timer: Timer,
+}
+
+impl FPSCounter {
+    /// Returns a new instance of the counter if requested by the user (via `SIXTYFPS_DEBUG_PERFORMANCE` environment variable).
+    pub fn new() -> Option<Rc<Self>> {
+        std::env::var("SIXTYFPS_DEBUG_PERFORMANCE").is_ok().then(|| {
+            Rc::new(Self { frame_times: Default::default(), update_timer: Default::default() })
+        })
+    }
+
+    /// Call this function if you want to start measurements. This will also print out some system information such as whether
+    /// this is a debug or release build, as well as the provided winsys_info string.
+    pub fn start(self: &Rc<Self>, winsys_info: &str) {
+        #[cfg(debug_assertions)]
+        let build_config = "debug";
+        #[cfg(not(debug_assertions))]
+        let build_config = "release";
+
+        eprintln!(
+            "sixtyfps: starting fps measurements. build config: {}; backend/windowing system: {}",
```

Comment:
```suggestion
            "SixtyFPS: Build config: {}; Backend: {}",
```

I like it shorter and to the point

Reply:
👍